aim/cli/convert/processors/wandb.py

```diff
@@ -0,0 +1,67 @@
+import click
+from aim import Run
+
+
+def parse_wandb_logs(repo_inst, entity, project, run_id):
+    try:
+        import wandb
+    except ImportError:
+        click.echo(
+            'Could not process wandb logs - failed to import "wandb" module.', err=True
+        )
+        return
+
+    client = wandb.Api()
+
+    if run_id is None:
+        # process all runs
+        runs = client.runs(entity + "/" + project)
+    else:
+        try:
+            # get the run by run_id
+            run = client.run(f"{entity}/{project}/{run_id}")
+        except Exception:
+            click.echo(f'Could not find run "{entity}/{project}/{run_id}"', err=True)
+            return
+        runs = (run,)
+
+    for run in runs:
+        if run.config.items():
+            aim_run = Run(
```

Comment:
we need to make sure no items are tracked in a background. the stdout/stderr tracking should be disabled as well.